src/arm/bbai-bone-buses.dtsi

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include <dt-bindings/board/am572x-bbai-pins.h>
+
+&dra7_pmx_core {
+	bone_uart_1_pins: pinmux_bone_uart_1_pins {
+		pinctrl-single,pins = <
+			P9_24( PIN_OUTPUT_PULLUP | MUX_MODE3)	/* gpio6_15.uart10_txd */
+			P9_26A( PIN_INPUT_PULLUP | MUX_MODE3)	/* gpio6_14.uart10_rxd */
+			/* unused pins */
+			P9_26B( PIN_OUTPUT | MUX_MODE15)
+		>;
+	};
+
+	bone_uart_2_pins: pinmux_bone_uart_2_pins {
+		pinctrl-single,pins = <
+			P9_21B( PIN_OUTPUT_PULLUP | MUX_MODE1)	/* spi2_d1.uart3_txd */
+			P9_22B( PIN_INPUT_PULLUP  | MUX_MODE1)	/* spi2_sclk.uart3_rxd */
+			/* unused pins */
+			P9_21A( PIN_OUTPUT | MUX_MODE15)
+			P9_22A( PIN_OUTPUT | MUX_MODE15)
+		>;
+	};
+
+	bone_uart_4_pins: pinmux_bone_uart_4_pins {
+        pinctrl-single,pins = <
```

Comment:
Watch your tabs!

Reply:
I am really very sorry, It looks fine on vscode here :(